src/NUnitEngine/mock-assembly-x86/mock-assembly-x86.csproj

```diff
@@ -5,6 +5,7 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
     <PlatformTarget>x86</PlatformTarget>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
If desired, this could be added in https://github.com/nunit/nunit-console/blob/master/src/Directory.Build.props rather than in each csproj.

Reply:
I felt like having it here gave better visibility. In fact, I'm not in love with Directory.Build.props but I haven't wanted to change it because some people like that better for things that are always the same - or which we hope will remain so. :-)